packages/block-editor/src/components/inserter/style.scss

```diff
@@ -222,73 +222,29 @@ $block-inserter-tabs-height: 44px;
 }
 
 .block-editor-inserter__category-tablist {
-	display: flex;
-	flex-direction: column;
-	border: none;
+	&::before {
+		background-color: var(--wp-admin-theme-color) !important;
+		opacity: 0.04;
+	}
+
 	margin-bottom: $grid-unit-10;
 	// Push the listitem wrapping the "explore" button to the bottom of the panel.
+	// TODO: is this outdated? The selected element doesn't seem to exist.
 	div[role="listitem"]:last-child {
 		margin-top: auto;
 	}
```

Comment:
Let's remove this, it's unnecessary since #60970.

Reply:
Done